### PR TITLE
Mpl2 change output prints

### PR DESCRIPTION
**Pull request by arthurjolo**
Solves [#3938](https://github.com/The-OpenROAD-Project/OpenROAD/issues/3938) .

Make reports that are not useful for users debug prints.

### CODE REVIEW

File: src/mpl2/src/hier_rtlmp.cpp [hunk 1]
```diff
@@ -260,9 +260,13 @@ void HierRTLMP::setDefaultThresholds()
     } else {
       congestion_weight_ = 1.0 * snap_layer_ / layers.size();
     }
-    logger_->report("snap_layer : {}  congestion_weight : {}",
-                    snap_layer_,
-                    congestion_weight_);
+    debugPrint(logger_,
+               MPL,
+               "macro_plamcement",
```
Comment (maliberty) [inline src/mpl2/src/hier_rtlmp.cpp:265]: Thoughout: macro_plamcement -> macro_placement

File: src/mpl2/src/hier_rtlmp.cpp [hunk 2]
```diff
@@ -420,17 +434,52 @@ void HierRTLMP::hierRTLMacroPlacer()
...
-  logger_->report("wirelength_weight_ = {}", wirelength_weight_);
-  logger_->report("guidance_weight_ = {}", guidance_weight_);
-  logger_->report("fence_weight_ = {}", fence_weight_);
-  logger_->report("boundary_weight_ = {}", boundary_weight_);
-  logger_->report("notch_weight_ = {}", notch_weight_);
-  logger_->report("macro_blockage_weight_ = {}", macro_blockage_weight_);
-  logger_->report("halo_width_ = {}", halo_width_);
-  logger_->report("halo_height_ = {}", halo_height_);
-  logger_->report("bus_planning_flag_ = {}", bus_planning_flag_);
+  debugPrint(
```
Comment (maliberty) [inline src/mpl2/src/hier_rtlmp.cpp:437]: For this long sequence of messages I would just use 
```
if (logger_->debugCheck(MPL, "macro_placement", 1)) {
  logger_>report....
  ...
}
```

File: src/mpl2/src/hier_rtlmp.cpp [hunk 3]
```diff
@@ -585,16 +647,19 @@ void HierRTLMP::hierRTLMacroPlacer()
...
+               "Determine shaping function for clusters -- Macro Tilings.\n");
     calClusterMacroTilings(root_cluster_);
 
     // create pin blockage for IO pins
     createPinBlockage();
     //
     // Perform macro placement in a top-down manner (pre-order DFS)
     //
-    logger_->report("Perform Multilevel macro placement...");
+    logger_->report("\nPerform Multilevel macro placement...");
```
Comment (maliberty) [inline src/mpl2/src/hier_rtlmp.cpp:662]: Why does this start with \n ?

File: src/mpl2/src/hier_rtlmp.cpp [hunk 4]
```diff
@@ -624,7 +689,7 @@ void HierRTLMP::hierRTLMacroPlacer()
   }
   cluster_map_.clear();
 
-  logger_->report("number of updated macros : {}", num_updated_macros_);
+  logger_->report("\nnumber of updated macros : {}", num_updated_macros_);
```
Comment (maliberty) [inline src/mpl2/src/hier_rtlmp.cpp:692]: same

**Review by maliberty — CHANGES_REQUESTED** [review]

File: src/mpl2/src/hier_rtlmp.cpp (hunk 3 above)
Comment (arthurjolo) [reply src/mpl2/src/hier_rtlmp.cpp:662]: No particular reason, but I felt easier to read the information on the log with an extra line separation this blocks

File: src/mpl2/src/hier_rtlmp.cpp (hunk 1 above)
Comment (arthurjolo) [reply src/mpl2/src/hier_rtlmp.cpp:265]: I followed some other debug prints that were already there. Should I create another group? If so, how can I do that? I looked but didn't found where I can create it.

Comment (maliberty) [reply src/mpl2/src/hier_rtlmp.cpp:265]: It just looks like a typo.  I don't think it is meant to be a separate group.

Comment (arthurjolo) [reply src/mpl2/src/hier_rtlmp.cpp:265]: Ah now I see the typo

**Review by maliberty — APPROVED** [review]

**Pull request merged**